po/gnucash-pot.cmake

```diff
@@ -20,7 +20,7 @@ execute_process(
                         --flag=g_snprintf:3:c-format
                         --flag=g_scanner_error:2:c-format
                         --flag=g_scanner_warn:2:c-format
-                        --keyword=_
+                        --keyword=G_
```

Comment:
You can't remove the `_` keyword. It's still in use by C and C++ code. Just add the `G_` keyword instead.